ddtrace/contrib/requests/patch.py

```diff
@@ -33,14 +34,22 @@ def _traced_request_func(func, instance, args, kwargs):
     # sessions to have their own (with the standard global fallback)
     tracer = getattr(instance, 'datadog_tracer', ddtrace.tracer)
 
+    distributed_tracing_enabled = getattr(instance, 'distributed_tracing_enabled', None)
```

Comment:
Since we don't have a configuration system in the Python client (something I want to design like we did for Ruby), can you provide more details about how people will use our `distributed_tracing_enabled`? Writing the example in the module docstring is enough.

Reply:
Yup, will do.